src/Components/PresentationalComponents/PaginationWrapper/PaginationWrapper.js

```diff
@@ -1,45 +1,36 @@
 
 import { Pagination, PaginationVariant } from '@patternfly/react-core';
 import propTypes from 'prop-types';
-import React, { Component } from 'react';
+import React from 'react';
 import { DEFAULT_PAGE_SIZE } from '../../../Helpers/constants';
 
-class PaginationWrapper extends Component {
-    static propTypes = {
-        apply: propTypes.func,
-        meta: propTypes.object,
-        variant: propTypes.string
-    };
+const PaginationWrapper = ({ apply, meta, variant }) => {
+    const { page, total_items: totalItems, page_size: pageSize } = meta;
 
-    static defaultProps = {
-        variant: 'bottom'
-    };
-    shouldComponentUpdate(nextProps) {
-        if (nextProps.meta === this.props.meta) {
-            return false;
-        }
+    const handleChangePage = (_event, pageNumber) => apply({ page: pageNumber });
+    const handleSetPageSize = (_event, perPage) => apply({ page_size: perPage, page: 1 });
 
-        return true;
-    }
-    handleChangePage = (_event, pageNumber) => this.props.apply({ page: pageNumber });
+    return (
+        <Pagination
+            page={page || 1}
```

Comment:
let's unify the fallback by using the `defaultProps` for these values (1, 0, DEFAULT_PAGE_SIZE)

Reply:
That is a good idea, but `page`, `totalItems`, `pageSize` are nested inside `meta`, [so it would not work](https://stackoverflow.com/a/38125155/4760478).

If I set
```
defaultProps = {
    meta: {
        page: 1,
        totalItems: 0,
        pageSize: DEFAULT_PAGE_SIZE
    }
}
```
and someone supplied only
```
meta: {
    page: 1
}
```
`meta` is defined so we would use this value above with missing properties.